SoftLayer/managers/network.py

```diff
@@ -137,15 +139,24 @@ def add_subnet(self, subnet_type, quantity=None, vlan_id=None, version=4,
                 desc = 'Global'
             elif subnet_type == 'public':
                 desc = 'Portable'
+            elif subnet_type == 'static':
+                desc = 'Static'
 
         # In the API, every non-server item is contained within package ID 0.
         # This means that we need to get all of the items and loop through them
         # looking for the items we need based upon the category, quantity, and
         # item description.
         price_id = None
         quantity_str = str(quantity)
-        for item in package.getItems(id=0, mask='itemCategory'):
+        # package_items = package.getItems(id=0, mask='itemCategory')
```

Comment:
if this code ( and the commented out block below) are not needed, just remove them.